pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/XPathPanelController.java

```diff
@@ -112,8 +120,70 @@ public void initialize(URL location, ResourceBundle resources) {
                            // Reevaluate XPath anytime the expression or the XPath version changes
                            .or(xpathVersionProperty().changes())
                            .subscribe(tick -> parent.refreshXPathResults());
+
+        xpathExpressionArea.plainTextChanges()
+                           .filter(t -> {
+                               if (t.getInserted().contains("/")) {
+                                   return xpathExpressionArea.getText().substring(xpathExpressionArea.getText().lastIndexOf("/", xpathExpressionArea.getText()
```

Comment:
This doesn't do at all what we want. You should use the position of the change...